src/styles/blocks/star.scss

```diff
@@ -0,0 +1,21 @@
+.card__stars {
+  display: flex;
+
+  &-star {
+    width: 16px;
+    height: 16px;
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
```

Comment:
Empty lines of code can confuse readers. It is recommended to remove the empty line.